### PR TITLE
feat: Hide area and symbol layers options if no matching geo dimensions are found

### PR DESCRIPTION
**Pull request by bprusinowski**

### CODE REVIEW

File: app/locales/de/messages.po [hunk 1]
```diff
@@ -13,19 +13,23 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: \n"
 
+#: app/configurator/components/chart-configurator.tsx:385
+msgid "Add filter"
+msgstr ""
```
Comment (ptbrowne) [inline app/locales/de/messages.po:18]: Missing trnslation ?

File: app/locales/de/messages.po [hunk 2]
```diff
@@ -498,11 +502,16 @@ msgstr "Interaktive Filter hinzufügen"
 msgid "controls.section.interactiveFilters.dataFilters"
 msgstr "Datenfilter"
 
+#: app/configurator/map/map-chart-options.tsx:154
+#: app/configurator/map/map-chart-options.tsx:341
+msgid "controls.section.map.noGeoDimensions"
+msgstr "In diesem Datenset können keine geografischen Dimensionen angezeigt werden!"
```
Comment (ptbrowne) [inline app/locales/de/messages.po:508]: Maybe we should not have a "!" at the end, I found it a bit "violent".

File: app/locales/de/messages.po (hunk 1 above)
Comment (ptbrowne) [inline app/locales/de/messages.po:18]:
```suggestion
msgstr "Filter hinzufügen"
```

@AnninaWalker ?

**Review by ptbrowne — APPROVED** [review]
There seem to be a missing translation in German (not sure it is from this PR though) and I am wondering about the exclamation point in the note about no geographical dimensions. Otherwise LGTM.

File: app/locales/de/messages.po (hunk 1 above)
Comment (ptbrowne) [reply app/locales/de/messages.po:18]: Fixed on main

Comment (bprusinowski) [reply app/locales/de/messages.po:18]: I have marked this PR as auto merge, but I will handle these translations in another PR, thanks!

Comment (ptbrowne) [reply app/locales/de/messages.po:18]: Solved on main

File: app/locales/de/messages.po (hunk 2 above)
Comment (bprusinowski) [reply app/locales/de/messages.po:508]: done in another PR 👍

**Pull request merged**